Clickstream/NetworkManager/Infrastructure/Sockets/SocketHandler.swift

```diff
@@ -217,3 +240,32 @@ extension SocketHandler {
         }
     }
 }
+
+extension DefaultSocketHandler {
+    func trackHealthEvent(eventName: HealthEvents,
```

Comment:
▿ Optional<Error>
  ▿ some : POSIXErrorCode: Connection reset by peer
    - posix : Darwin.POSIXErrorCode

I got this error but it did not go inside any of the if cases and got returned without sending an event.

Reply:
Please help me in replicating this